rpms/generate_rpm_package.sh

```diff
@@ -61,12 +61,12 @@ build_rpm() {
         VERSION="$(cat ${SOURCES_DIRECTORY}/src/VERSION | cut -d 'v' -f 2)"
         echo "Version is $VERSION"
         if [[ "${TARGET}" == "manager" ]] && [[ "${LEGACY}" == "yes" ]]; then
-            MAJOR_MINOR="$(echo ${VERSION} | cut -d "v" -f 2)"
+            MAJOR_MINOR="$(echo ${VERSION}|cut -d . -f 1-2)"
```

Comment:
This won't ever work. If `VERSION` is `v3.10.0`, for example, line 64 generates this: `MAJOR_MINOR = v3.10` and the `if` from 66 won't work. 

To extract the major minor, you must run this: `echo ${VERSION} | cut -d v -f 2 | cut -d . -f 1-2`

Reply:
The `v` from the version is removed at line 61 so it should not be necessary to remove it again.